### PR TITLE
Remove unnecessary `throws` declaration

### PR DESCRIPTION
**Pull request by xnull**
## Overview

Description:
Remove unnecessary `throws` declaration

Why should this be merged: 

Related issue(s) (if applicable): #<number>

## Checklist (Definition of Done):

- [ ] There are no TODOs left in the code
- [ ] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [ ] Change is covered by automated tests
- [ ] Public API has Javadoc

### CODE REVIEW

**Comment by xnull** [comment]
@lixinchengdu it's dead code, it's annoying. IDEA makes it yellow. So, that's the reason. 
Those method signatures can confuse also.

**Comment by annym** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651466&bid=15577779)

File: runtime/src/main/java/org/corfudb/runtime/clients/LogUnitHandler.java [hunk 1]
```diff
@@ -83,8 +82,7 @@ private static Object handleTrimmed(CorfuMsg msg, ChannelHandlerContext ctx, ICl
      * @throws OverwriteException Throws OverwriteException if address has already been written to.
      */
     @ClientHandler(type = CorfuMsgType.ERROR_OVERWRITE)
-    private static Object handleOverwrite(CorfuPayloadMsg<Integer> msg, ChannelHandlerContext ctx, IClientRouter r)
-            throws Exception {
+    private static Object handleOverwrite(CorfuPayloadMsg<Integer> msg, ChannelHandlerContext ctx, IClientRouter r) {
```
Comment (annym) [inline runtime/src/main/java/org/corfudb/runtime/clients/LogUnitHandler.java]: Codacy Issue found: [Avoid unused method parameters such as 'r'.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651466)

File: runtime/src/main/java/org/corfudb/runtime/clients/LogUnitHandler.java [hunk 2]
```diff
@@ -126,8 +123,7 @@ private static Object handleValueAdoptedResponse(CorfuPayloadMsg<ReadResponse> m
      * @throws OutOfSpaceException Throws OutOfSpaceException if log unit out of space.
      */
     @ClientHandler(type = CorfuMsgType.ERROR_OOS)
-    private static Object handleOos(CorfuMsg msg, ChannelHandlerContext ctx, IClientRouter r)
-            throws Exception {
+    private static Object handleOos(CorfuMsg msg, ChannelHandlerContext ctx, IClientRouter r) {
```
Comment (annym) [inline runtime/src/main/java/org/corfudb/runtime/clients/LogUnitHandler.java]: Codacy Issue found: [Avoid unused method parameters such as 'r'.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651466)

Comment (annym) [inline runtime/src/main/java/org/corfudb/runtime/clients/LogUnitHandler.java]: Codacy Issue found: [Avoid unused private methods such as 'handleOos(CorfuMsg,ChannelHandlerContext,IClientRouter)'.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651466)

File: runtime/src/main/java/org/corfudb/runtime/clients/LogUnitHandler.java (hunk 1 above)
Comment (annym) [inline runtime/src/main/java/org/corfudb/runtime/clients/LogUnitHandler.java]: Codacy Issue found: [Avoid unused private methods such as 'handleOverwrite(CorfuPayloadMsg,ChannelHandlerContext,IClientRouter)'.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651466)

File: runtime/src/main/java/org/corfudb/protocols/wireprotocol/ClientHandshakeHandler.java [hunk 3]
```diff
@@ -181,11 +178,10 @@ public void channelInactive(ChannelHandlerContext ctx) throws Exception {
      *
      * @param ctx channel handler context
      * @param cause exception cause
-     * @throws Exception
      */
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx,
-        java.lang.Throwable cause) throws Exception {
+        java.lang.Throwable cause) {
```
Comment (annym) [inline runtime/src/main/java/org/corfudb/protocols/wireprotocol/ClientHandshakeHandler.java]: Codacy Issue found: [Unnecessary use of fully qualified name 'java.lang.Throwable' due to existing implicit import 'java.lang.*'](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651466)

**Comment by annym** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/CorfuDB/CorfuDB/pullRequest?prid=4653438&bid=15584329)

**Pull request closed without merging**